app/plugin_api.go

```diff
@@ -285,6 +285,10 @@ func (api *PluginAPI) SearchChannels(teamId string, term string) (*model.Channel
 	return api.app.SearchChannels(teamId, term)
 }
 
+func (api *PluginAPI) SearchUsers(search *model.UserSearch, options *model.UserSearchOptions) ([]*model.User, *model.AppError) {
```

Comment:
You need to change this in order to implement the API method. This is also by the build failed.
```suggestion
func (api *PluginAPI) SearchUsers(search *model.UserSearch) ([]*model.User, *model.AppError) {

```

Reply:
updated